api/lib/source/cosmosV0-source.js

```diff
@@ -288,6 +290,66 @@ class CosmosV0API extends RESTDataSource {
     )
   }
 
+  async getRecentProposals() {
```

Comment:
Isn't this a duplicate of `getAllProposals`?

Reply:
I guess we can just remove this function since it is not doing much